assets/js/payment-method-extensions/payment-methods/stripe/stripe-utils/type-defs.js

```diff
@@ -277,18 +277,23 @@
 /**
  * @typedef {Object} StripeServerData
  *
- * @property {string}              stripeTotalLabel The string used for payment descriptor.
- * @property {string}              publicKey        The public api key for stripe requests.
- * @property {boolean}             allowPrepaidCard True means that prepaid cards can be
- *                                                  used for payment.
- * @property {Object}              button           Contains button styles
- * @property {string}              button.type      The type of button.
- * @property {string}              button.theme     The theme for the button.
- * @property {string}              button.height    The height (in pixels) for the button.
- * @property {string}              button.locale    The locale to use for stripe elements.
- * @property {boolean}             inline_cc_form   Whether stripe cc should use inline cc
- *                                                  form or separate inputs.
- * @property {{[k:string]:CCIcon}} icons            Contains supported cc icons.
+ * @property {string}                      stripeTotalLabel The string used for payment
+ *                                                          descriptor.
+ * @property {string}                      publicKey        The public api key for stripe
+ *                                                          requests.
+ * @property {boolean}                     allowPrepaidCard True means that prepaid cards
+ *                                                          can be used for payment.
+ * @property {Object}                      button           Contains button styles
+ * @property {string}                      button.type      The type of button.
+ * @property {string}                      button.theme     The theme for the button.
+ * @property {string}                      button.height    The height (in pixels) for
+ *                                                          the button.
+ * @property {string}                      button.locale    The locale to use for stripe
+ *                                                          elements.
+ * @property {boolean}                     inline_cc_form   Whether stripe cc should use
+ *                                                          inline cc
+ *                                                          form or separate inputs.
+ * @property {{[k:string]:CreditCardIcon}} icons            Contains supported cc icons.
```

Comment:
Thanks for doing this, and taking care of the spacing too. I appreciate the effort 👏

Reply:
I know we usually wrap at 80, but I remember you increased the limit to 100?

Reply:
> but I remember you increased the limit to 100?

Well it's not a hard increase (comment blocks aren't currently linted), I just added another column guide in my IDE for 100 so for comment blocks I could use that as a guide if 80 was too short.  Doing it on 90 here was fine :)